product_supplierinfo_tree_price_info/demo/res_groups.yml

```diff
@@ -0,0 +1,37 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015-Today GRAP (http://www.grap.coop)
+#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+- !record {model: res.groups, id: base.group_no_one}:
```

Comment:
@pedrobaeza : Thanks for your review again ! You're on every front !

1/ the objective of demo data (**for me**) is to provide full testable environment: 
* for users. (developper or functional). The aim : "One click and you can test on runbot". So, for that, I allways set admin into all the needed groups to see correctly how the module works without any extra settings;
* for computer, providing data that will be available for automatic tests ;

2/ I put it in the other file ./product_supplierinfo.yml. Or maybe, you're talking about other things...